projects/project-player-library/src/lib/shared/subtask-card/subtask-card.component.ts

```diff
@@ -70,4 +76,24 @@ setOptionList(){
   let options:any = actions.TASK_STATUS;
   this.subTaskOptions = options;
 }
+onDateChange(newDate: Date) {
```

Comment:
Refer to the comment made in the task-details-page.component.ts file and make relevant changes

Reply:
done